client/reader/following/style.scss

```diff
@@ -169,3 +169,25 @@
 		width: 20px;
 	}
 }
+
+.following__view-toggle {
+	display: flex;
+	gap: 8px;
+	
+	.components-button {
+		height: 36px;
+		min-width: 36px;
+		padding: 6px;
+		color: var(--color-text);
+		
+		&.is-pressed {
+			background: var(--color-text);
+			color: var(--color-surface);
+		}
+		
+		&:hover {
+			background: var(--color-text);
+			color: var(--color-surface);
+		}
+	}
+}
```

Comment:
It would be better to move these styles to `following/view-toggle.scss`.

Reply:
Since this toggle is temporary and lives in the following/ directory, I'd prefer to keep the styles here for now. Do you feel strongly about putting them in a separate file? Is there a reason I'm missing?

Reply:
No strong opinion. I have suggested based on our [CSS Coding Guidelines](https://github.com/Automattic/wp-calypso/blob/trunk/docs/coding-guidelines/css.md).